modules/modify-policy-categories-using-tab.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-security-policies.adoc
+:_content-type: PROCEDURE
+[id="modify-policy-categories-using-tab_{context}"]
+= Modifying policy categories using the Policy categories tab
+
+{product-title-short} version 3.74 provides a new method to create and manage policy categories in {product-title-managed} or in {product-title-short} if you have the PostgreSQL database (Technology Preview) enabled. All policy workflows other than policy creation remain unchanged when using this feature. For instructions on using this feature, see the second procedure in the following section.
+
+You can also configure policy categories by using the `PolicyCategoryService` API object. For more information, navigate to *Help* -> *API reference* in the {product-title-short} portal. 
+
+:FeatureName: PostgreSQL support
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+
+To edit or delete policy categories in {product-title-managed} or if you have the PostgreSQL database feature (Technology Preview) enabled:
```

Comment:
@kcarmichael08 I agree with the change that Andrea suggested about the lead-in sentence, did you want to move it before merging?

Reply:
Oops, I thought I got that! Yes, let me fix. :)

Reply:
@kcarmichael08 Lmk when you're ready for this to be merged! Thank ya!

Reply:
Sorry, I was waiting for the preview to rebuild - I think it's good to go though!